pkg/network/ebpf/c/protocols/kafka/types.h

```diff
@@ -121,6 +121,10 @@ typedef struct {
     // The array topic_name_size_buckets maps a bucket index to the number of occurrences observed for topic name lengths
     __u64 topic_name_size_buckets[KAFKA_TELEMETRY_TOPIC_NAME_NUM_OF_BUCKETS];
     __u64 produce_no_required_acks;
+
+    // Arrays to keep track of the number of occurrences of each API version that were classified as kafka, before decoding
+    __u64 classified_fetch_api_version_hits[KAFKA_TELEMETRY_API_VERSIONS_NUM_OF_BUCKETS];
+    __u64 classified_produce_api_version_hits[KAFKA_TELEMETRY_API_VERSIONS_NUM_OF_BUCKETS];
```

Comment:
can reuse the already existing consts for max-<produce/fetch>-for-classification